contracts/mocks/testnet-helpers/Faucet.sol

```diff
@@ -14,10 +15,12 @@ contract Faucet is IFaucet, Ownable {
   // If disabled, anyone can call mint at the faucet, for PoC environments
   bool internal _permissioned;
 
+  // Maximum amount of tokens per mint allowed
+  uint256 private constant MAX_MINT_AMOUNT = 10000;
```

Comment:
we can make it public

Reply:
`5166eb05400757d705f83ffb61295fc2c85553d8`